src/operator/tensor/elemwise_binary_broadcast_op.h

```diff
@@ -144,7 +144,7 @@ void BinaryBroadcastCompute(const nnvm::NodeAttrs& attrs,
   int ndim = BinaryBroadcastShapeCompact(inputs[0].shape_, inputs[1].shape_, outputs[0].shape_,
                                          &new_lshape, &new_rshape, &new_oshape);
   if (!ndim) {
-    BinaryCompute<xpu, OP>(attrs, ctx, inputs, req, outputs);
+    ElemwiseBinaryOp::Launch<xpu, OP>(attrs, ctx, inputs, req, outputs);
```

Comment:
Since `Launch` has the `ElemwiseBinaryOp ` qualifier, I think this is fine. @reminisce